cfgen/cfgen

```diff
@@ -854,38 +854,44 @@ ProcT.__doc__ = 'Type of process'
 
 class PortalGroup(IntEnum):
     hax = 22000,
+    hax_http = 8008,
```

Comment:
Why do we need http port to be part of PortalGroup? I think we need not have it as it will not be used for motr endpoint creation.

Reply:
Dropped `hax_http` from PortalGroup and updated related code.